src/modding/ModManager.cs

```diff
@@ -826,6 +1238,13 @@ private void AvailableModSelected(int index)
 
         if (enabledModsContainer.IsAnythingSelected())
             enabledModsContainer.UnselectAll();
+
+        leftArrow.Disabled = true;
+        rightArrow.Disabled = false;
+        modErrorsContainer.UnselectAll();
+        errorInfoLabel.Hide();
+        moveModUpButton.Disabled = availableModsContainer.IsSelected(0);
+        moveModDownButton.Disabled = availableModsContainer.IsSelected(availableModsContainer.GetItemCount() - 1);
```

Comment:
These 100% need to be put in a separate method to reduce this complex bit of code duplication.